internal/pkg/remote/endpoint/service.go

```diff
@@ -123,16 +123,20 @@ func (config *Config) GetAllServices() (map[string][]Service, error) {
 		Timeout: defaultTimeout,
 	}
 
-	url := "https://" + config.URI + "/assets/config/config.prod.json"
+	epURL, err := config.GetURL()
+	if err != nil {
+		return nil, err
+	}
 
-	req, err := http.NewRequest(http.MethodGet, url, nil)
+	configURL := epURL + "/assets/config/config.prod.json"
+	req, err := http.NewRequest(http.MethodGet, configURL, nil)
```

Comment:
This I think is the cause of the e2e failures.  It should not be looking up that file when the remote is set to not be active.

Reply:
But then, the problem already exists before my patch. The only thing the patch changes is the scheme in the url. Before the URL was requested, no matter if the remote was active or not. 

The errors in the ci are:
```
key.go:143: 
        "/usr/local/bin/apptainer key search -u https://keys.sylabs.io/ 7605BC2716168DF057D6C600ACEEC62C8BD91BEE" output:
        FATAL:   Keyserver client failed: error making request to server: Get "https://cloud.apptainer.org/assets/config/config.prod.json": dial tcp: lookup cloud.apptainer.org: no such host
´´´
which means that the DNS lookup failed and nothing with the request itself. If I execute the same statement on my local machine with my branch I get:
```
$ apptainer version
1.0.2+345-gebf5060be
$ apptainer key search -u https://keys.sylabs.io 0x7605BC2716168DF057D6C600ACEEC62C8BD91BEE
Showing 1 results

KEY ID    BITS  NAME/EMAIL
8BD91BEE  4096  WestleyK (Testing key; used for signing test containers) <westley@sylabs.io>
```
So this seems to be an infrastructure problem of the test environment.